src/ari_client.py

```
@@ -728,6 +728,59 @@ async def record_channel(
             logger.error("Error starting ARI channel recording", channel_id=channel_id, exc_info=True)
             return False
 
+    async def record_bridge(
+        self,
+        bridge_id: str,
+        name: str,
+        format: str = "wav",
```

Ruff (0.15.21)

[error] Function argument `format` is shadowing a Python builtin

(A002)